### PR TITLE
Update release script

### PR DESCRIPTION
**Pull request by anirvanmandal**
* Update the release Rakefile; Update Bump to v0.9
* Add Changelog

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -31,16 +31,42 @@ namespace :release do
 
   desc 'bump version'
   task :bump, [:type] do |_, args|
-    Bump::Bump.run(args[:type], commit: true, bundle: true, tag: true)
+    Bump.replace_in_default = ['README.md']
+    Bump.changelog = true
+    Rake::Task['release:generate_commit_message'].invoke
+    Bump::Bump.run(args[:type], commit: true, bundle: true, tag: false, commit_message: "\n\n#{ENV['COMMIT_MESSAGE']}")
```
Comment (jasonplumb) [inline Rakefile:37]: @anirvanmandal am I correct in thinking we have to set ENV['COMMIT_MESSAGE'] to be the contents of the changelog entry? Perhaps an upgrade note is needed within the changelog.md to explain if anything has to be set

**Review by jasonplumb — APPROVED** [review]
approving but I have a question to help with respect to ENV['COMMIT_MESSAGE'] usage

File: Rakefile (hunk 1 above)
Comment (anirvanmandal) [reply Rakefile:37]: The ENV variable is the changelog entries added to the unreleased section.

Comment (jasonplumb) [reply Rakefile:37]: So on heroku we do something like ENV['COMMIT_MESSAGE'] = 'summary of entry [ id ] (link)'? and need to remember to update the comment for each release?

**Pull request merged**